### ISSUE TITLE
Manually adding subtitles doesn't work.

### ISSUE THREAD

**Issue by denismattos** [mrknow/filmkodi#490]
I'm having trouble with manually added subtitles. 
After I click the audio button and then "Browse for subtitles", the browsing "card" shows up, but the "Working..." message just goes on forever and no folders are shown.

Searching the internet for a solution, I found a 2012 thread of the [official forum](http://forum.kodi.tv/) in which [Martijn](https://kodi.tv/author/martijn/) [revealed the team is actually aware of this bug, but deliberately didn't solve it to avoid associating themselves with piracy](http://forum.kodi.tv/showthread.php?tid=156801&pid=1476371#pid1476371) (since they perceive streaming addons as something "99,9% of (...) users use (...) for pirated content"). So – apparently – this is a bug we can't hope to see solved Kodi-side.  
Also: I can't guarantee, but I have the distinct impression this started happening after Specto's last update.  
([This is a crosspost from /r/Addons4Kodi](https://www.reddit.com/r/Addons4Kodi/comments/52owai/browse_for_subtitles_doesnt_work/).)

**Comment by denismattos** [comment]
For anyone experiencing the same issue: I found a workaround.
I added my Kodi subtitles folder to my library (under Videos, but I surmise it would've been the same under Music). When I open the "Browse for subtitles..." option, I right click, "cancelling" the card. A "Movie and alternate subtitle directory" option instantly – no "Working..." message – appears, along with my library folders. As my subtitles folder is now among those, I simply open it and click the subtitle file.
Done.

So now there's a provisional solution while we wait for the developer's answer on this issue. =)
